src/Algebra/Properties/Semiring.agda

```diff
@@ -0,0 +1,33 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some basic items and theory for Semiring.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Semiring)
+import Algebra.Properties.Monoid.Divisibility as MonoidDiv
+open import Data.Product using (_,_)
+
+module Algebra.Properties.Semiring {r₁ r₂} (R : Semiring r₁ r₂) where
```

Comment:
This should be `Algebra.Properties.Semiring.Divisibility`

Reply:
Done.